Portfolio1.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en-GB">
```

Comment:
rename to index.html as is standard practice

Reply:
Hi Mike, I'll definitely keep this in mind for next time, but right now it messes up git if I change the name and I need to get this to Rayna, so i'll have to leave this one for now.